deploy/kubernetes-manifests/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    logz.io/skip: "true"
+  labels:
+    app: kubernetes-instrumentor
+  name: kubernetes-instrumentor
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubernetes-instrumentor
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        app: kubernetes-instrumentor
+    spec:
+      containers:
+      - args:
+          - --secure-listen-address=0.0.0.0:8443
+          - --upstream=http://127.0.0.1:8080/
+          - --logtostderr=true
+          - --v=0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
+        name: apis-rbac-proxy
+        ports:
+          - containerPort: 8443
+            name: https
+            protocol: TCP
+      - args:
+          - --health-probe-bind-address=:8081
+          - --metrics-bind-address=127.0.0.1:8080
+          - --leader-elect
+          - --instrumentation-detector-tag=v1.0.8-b
+          - --instrumentation-detector-image=logzio/instrumentation-detector
+        command:
+          - /app
+        image: "logzio/instrumentor:v1.0.8-b"
+        env:
+          - name: MONITORING_SERVICE_ENDPOINT
+            value: "logzio-monitoring-otel-collector.monitoring.svc.cluster.local"
+          - name: DOTNET_AGENT_IMAGE
+            value: "logzio/otel-agent-dotnet:v0.0.1-test"
+          - name: JAVA_AGENT_IMAGE
+            value: "logzio/otel-agent-java:v0.0.1-test"
+          - name: NODEJS_AGENT_IMAGE
+            value: "logzio/otel-agent-nodejs:0.0.6-test"
+          - name: PYTHON_AGENT_IMAGE
+            value: "logzio/otel-agent-python:0.0.8-test"
```

Comment:
Will we still use tags with `-test` when we'll release it?

Reply:
No :) we will create official logzio images